examples/FW-APSP/FloydIterativeKernelDF.h

```diff
@@ -0,0 +1,24 @@
+#ifndef FLOYD_ITERATIVE_KERNEL
+#define FLOYD_ITERATIVE_KERNEL
+
+#include <algorithm>    /* std::min */
+#include <omp.h>
+using namespace std;
+
+template <typename T>
+BlockMatrix<T> floyd_iterative_kernel(int block_size, BlockMatrix<T> m_ij, 
+							BlockMatrix<T> m_ik, BlockMatrix<T> m_kj) {
+	//BlockMatrix<T> bm(block_size, block_size);
+  for(int k = 0; k < block_size; ++k) {
+		//#pragma ivdep
+    for(int i = 0; i < block_size; ++i) {
+			for(int j = 0; j < block_size; ++j) {
+				m_ij(i,j,min(m_ij(i,j), m_ik(i,k) + m_kj(k,j)));
```

Comment:
This might be a valid statement, but it has no outcome right ?

Reply:
This is a valid statement and it modifies the element inside the block matrix, but I changed the operator overloading to make it more intuitive. This change is part of the pull request #61